### PR TITLE
Add new test for empty observables

### PR DESCRIPTION
**Pull request by YevhenLysen**

### CODE REVIEW

**Review by nonspher — APPROVED** [review]

File: tests/functional/tests/test_empty_observables.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+from ctrlibrary.threatresponse.enrich import enrich_observe_observables
+from tests.functional.tests.constants import MODULE_NAME
+
+
+@pytest.mark.parametrize(
+    'observable_type, observable',
+    (
+     ('ip', '4.3.1.4'),
+     ('domain', 'wp.org'),
+     ('md5', 'd41d8cd98f00b204e9800998ecf8427e')
```
Comment (mistergreen22) [inline tests/functional/tests/test_empty_observables.py:12]: Hi please take a look to the use case - https://github.com/threatgrid/integrations/blob/master/Google%20Chronicle/Use_Case_Chronicle.md#input-observables
If have haven't all malicious observables types, the empty observables we always have. So please add all types (add ipv6, sha1, sha256) to this test.

**Review by mistergreen22 — APPROVED** [review]
